rules/symfony-code-quality/tests/Rector/Class_/EventListenerToEventSubscriberRector/Fixture/multiple_listeners_called_once.php.inc

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Rector\SymfonyCodeQuality\Tests\Rector\Class_\EventListenerToEventSubscriberRector\Fixture;
+
+class MultipleMethodsCalledOnce
```

Comment:
It need a name based on file name `multiple_listeners_called_once.php.inc` -> `class MultipleListenersCalledOnce`.

There is `validate-fixture-classname` command that can automatically fix that, but currently not runs for `rules` directory due many false positive.

```suggestion
class MultipleListenersCalledOnce
```